charmcraft.yaml

```diff
@@ -90,11 +89,275 @@ config:
+        
+        These credentials must have Secret Manager Read/Write access. Alpha support.
+      type: string
+
+    vault-address:
+      description: Defines the vault address to read/write Airbyte Configuration to Hashicorp Vault. Alpha Support.
+      type: string
+
+    vault-prefix:
+      description: Defines the vault path prefix. Empty by default. Alpha Support.
+      type: string
+
+    vault-auth-token:
+      description: The token used for vault authentication. Alpha Support.
+      type: string
+
+    vault-auth-method:
+      description: How vault will perform authentication. Currently, only supports Token auth. Defaults to "token". Alpha Support.
+      default: "token"
+      type: string
+
+    aws-access-key:
+      description: Defines the aws_access_key_id from the AWS credentials to use for AWS Secret Manager.
+      type: string
+
+    aws-secret-access-key:
+      description: Defines aws_secret_access_key to use for the AWS Secret Manager.
+      type: string
+
+    aws-kms-key-arn:
+      description: Optional param that defines the KMS Encryption key used for the AWS Secret Manager.
+      type: string
+
+    aws-secret-manager-secret-tags:
+      description: |
+        Defines the tags that will be included to all writes to the AWS Secret Manager.
+        
+        The format should be "key1=value1,key2=value2".
+      type: string
+
+    ##### Jobs config #####
+    sync-job-retries-complete-failures-max-successive:
+      description: Max number of successive attempts in which no data was synchronized before failing the job.
+      default: 5
+      type: int
+
+    sync-job-retries-complete-failures-max-total:
+      description: Max number of attempts in which no data was synchronized before failing the job.
+      default: 10
+      type: int
+
+    sync-job-retries-complete-failures-backoff-min-interval-s:
+      description: Minimum backoff interval in seconds between failed attempts in which no data was synchronized.
+      default: 10
+      type: int
+
+    sync-job-retries-complete-failures-backoff-max-interval-s:
+      description: Maximum backoff interval in seconds between failed attempts in which no data was synchronized.
+      default: 1800
+      type: int
+
+    sync-job-retries-complete-failures-backoff-base:
+      description: Exponential base of the backoff interval between failed attempts in which no data was synchronized.
+      default: 3
+      type: int
+
+    sync-job-retries-partial-failures-max-successive:
+      description: Max number of successive attempts in which some data was synchronized before failing the job.
+      default: 1000
+      type: int
+
+    sync-job-retries-partial-failures-max-total:
+      description: Max number of attempts in which some data was synchronized before failing the job.
+      default: 20
+      type: int
+
+    sync-job-max-timeout-days:
```

Comment:
What does this default to if no value is provided? Could be worth either adding this to the description if the answer is never, or adding the default value as a default here so it's clear.

Reply:
I've updated to add the same defaults as per upstream.